### PR TITLE
added --params / -p to ff ( findfile ) 

### PR DESCRIPTION
**Pull request by punixcorn**
## --params / -p 
With --params / -p you can pass normal `find` parameters to ff 
> the command :
```sh 
sudo find /home "*tempfile*" -delete -executable 2>/dev/null
```
> can be done with ff: 
```sh 
ff /home *tempfile* -p -delete -executable 
```

### CODE REVIEW

File: .src/ff.c [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+  } else {
+    fprintf(stderr,
+            "ERR: invalid arguments passed\n"
+            "Usage: %s [ path ] [ file/directory ] [ params...  ]\n",
+            exe_name);
+    exit(1);
+  }
+  putsinto(command, 3, "' ", temp, " 2> /dev/null");
+  printf("%s\n", command);
+  exit(0);
```
Comment (punixcorn) [inline .src/ff.c]: fuck

**Review by SpeedyQweku — APPROVED** [review]

**Comment by punixcorn** [comment]
bruh it not gonnna work

**Comment by SpeedyQweku** [comment]
Like how

**Comment by punixcorn** [comment]
ok fixed
added a -v to see the running command

**Comment by punixcorn** [comment]
> Like how

```c
printf("%s\n",command);
exit(0);
```
it doesnt even run the command

**Review by punixcorn — COMMENTED** [review]
looks ok

**Pull request merged**